xbmc/windowing/osx/OpenGL/WindowControllerMacOS.mm

```diff
@@ -53,6 +53,13 @@ - (nullable instancetype)initWithTitle:(NSString*)title defaultSize:(NSSize)size
   return self;
 }
 
+- (void)setGUIRenderingForPerformance:(BOOL)enabled
```

Comment:
Nitpick: i would remove “ForPerformance” from the function name. A method name should reflect what the method does, not why it is called.

Reply:
@rfarina51 what is your thought on this? Do you want to change that?